tailcfg/derpmap.go

```diff
@@ -92,6 +92,10 @@ type DERPRegion struct {
 	// "San Francisco", "Singapore", "Frankfurt", etc.
 	RegionName string
 
+	// RegionVendor is the name of the hosting provider for the region:
+	// "NetActuate", "Digital Ocean", "Hetzner", etc.
```

Comment:
Since we link these docs to end users for custom DERP maps, it might be worth mentioning that this field is only used for internal purposes and not exposed over the wire.

I assume it won't be in the final DERPMap endpoints?

Maybe it even needs to be `json:"-"` and of type `any` and named like:

```go
     InternalNotes any `json:"-"`
```

So we can put whatever there internally in corp.